### DISCUSSION TITLE
Relation Manager not show up on tenant mode

### DISCUSSION THREAD

**Discussion by widiu7omo** [filamentphp/filament#8378] [Help]
Hello, is there any guide how to show relation manager to edit resource when using tenant mode?

 
![Screen Shot 2023-09-10 at 10 45 04](https://github.com/filamentphp/filament/assets/25941348/0bda7e4c-9974-4c84-8f98-e028d8688687)

I just tried but the relation is not show up.

Thank You.

**Comment by danharrin** [comment]
There is no difference in tenant mode. Ensure that you have viewAny access to the related policy.

**Comment by micraux** [comment]
Did you set the `read-ony mode`?
https://filamentphp.com/docs/3.x/panels/resources/relation-managers#read-only-mode

I struggled a bit on that one because it is located at the end of the `Managing Relationship` section in the documentation.
